package.json

```diff
@@ -2,7 +2,8 @@
   "name": "aquarius-ts-capture-trigger",
   "description": "Deployment via serverless",
   "dependencies": {
-    "serverless": "1.77.1",
+    "serverless": "2.1.1",
+    "serverless-pseudo-parameters": "2.5.0",
```

Comment:
Just clarifying here - does `serverless-psuedo-parameters` need to be specified here or does it come along with serverless now? I remember a week or so ago we thought maybe it was just bundled with serverless.

Reply:
It does. Perhaps our investigation last week discovered spurious data.